pysen/ext/mypy_wrapper.py

```diff
@@ -190,9 +190,9 @@ class MypyTarget:
 @functools.lru_cache(1)
 def _check_mypy_version() -> None:
     version = get_version("mypy")
-    if version.major != 0 or version.minor < 770:
+    if version.major >= 2 or version.major == 0 and version.minor < 770:
```

Comment:
`version.major >= 2 or version.major == 0 and version.minor < 770`  == `version.major >= 2 or (version.major == 0 and version.minor < 770)`

https://docs.python.org/3/reference/expressions.html#operator-precedence